pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -5,6 +5,8 @@ import isEmpty from 'lodash/isEmpty';
 import Vue from 'vue';
 import { mapGetters } from 'vuex';
 
+import { escapeHtml } from '../utils/string';
```

Comment:
We usually import via `@pkg/…`, but I'll let this one slide to get things merged.